bin/create_json.py

```diff
@@ -18,36 +18,41 @@
 # path to the datadictionary/gdcdictionary/schemas/ folder
 shema_path: str = os.environ.get('SCHEMA_PATH', '../../gdcdictionary/schemas/')
 
-table_name_mapping = {"lesion_characteristic": "lesion_characteristics", "adverse_event": "adverse_events", "lab": "laboratory_test", "person": "demographics", "disease_characteristic": "disease_characteristics", "tumor_assessment": "tumor_characteristics", "biopsy_surgical_procedure": "biopsy_surgical_procedures", "survival_characteristic": "survival_characteristics", "myeloid_sarcoma_involvement": "disease_characteristics", "total_dose": "medication", "molecular_analysis": "genetic_analysis", "study": "subject_characteristics", "off_protocol_therapy_study": "off_protocol_therapy_or_study", "protocol_treatment_modification": "protocol_treatment_modifications", "subjects": "subject_characteristics", "late_effect": "late_effects"}
+table_name_mapping = {"lesion_characteristic": "lesion_characteristics", "adverse_event": "adverse_events", "lab": "laboratory_test", "person": "demographics", "disease_characteristic": "disease_characteristics", "tumor_assessment": "tumor_characteristics", "biopsy_surgical_procedure": "biopsy_surgical_procedures", "survival_characteristic": "survival_characteristics",
+                      "myeloid_sarcoma_involvement": "disease_characteristics", "total_dose": "medication", "molecular_analysis": "genetic_analysis", "subject": "subject_characteristics",  "study": "subject_characteristics", "off_protocol_therapy_study": "off_protocol_therapy_or_study", "protocol_treatment_modification": "protocol_treatment_modifications", "subject": "subject_characteristics", "late_effect": "late_effects"}
```

Comment:
It looks like the "subject": "subject_characteristics" key:value pair is now repeated  where before it was two distinct k:v pairs having the same value "subject_characteristics" and keys "subject" and "subjects". The second k:v pair should just override the first but if the intention is to remove the "subjects" entry then it would be less confusing to remove the duplicate entry.

Reply:
Good catch, not sure how it got duplicated, the intention was to change "subjects" -> "subject" since there does not seem to be a "subjects" schema table